projects/plugins/super-cache/inc/delete-cache-button.php

```diff
@@ -86,6 +86,17 @@ function wpsc_admin_bar_delete_cache_ajax() {
 		if ( defined( 'WPSCDELETEERROR' ) ) {
 			return json_decode( constant( 'WPSCDELETEERROR' ) );
 		} else {
+			$req_path = !empty($_POST['path']) ? $_POST['path'] : '/';
+			$referer = !empty($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : admin_url();
+			/**
+			 * Hook into the cache deletion process after a successful cache deletion from the admin bar button.
+			 *
+			 * @since 1.12
```

Comment:
@donnchawp maybe different version should be here?

Reply:
Yeah, can you change that version to this: `$$next-version$$`